modules/hosted-control-planes-concepts-personas.adoc

```diff
@@ -13,7 +13,7 @@ When you use hosted control planes for {product-title}, it is important to under
 [id="hosted-control-planes-concepts_{context}"]
 == Concepts
 
-hosted cluster:: An {product-title} API endpoint for the tenant cluster that is managed by the HyperShift Operator.
+hosted cluster:: An {product-title} API endpoint for the tenant cluster that is managed by the HyperShift Operator. The hosted cluster includes both the control plane and its corresponding data plane.
```

Comment:
This addresses one important question, does Hosted Cluster encompass cp and dp? but it does not solve the use of "tenant cluster" as yet another way to describe the concert. 

Also, I think using "an API endpoint" forces us to use yet another term to describe what a Hosted Cluster is. 

> ‘Hosted Cluster’ refers to an OpenShift cluster with its control-plane hosted on a distinct OpenShift cluster, known as the management cluster.”

Eventually we want to use "Hosted Cluster" to describe the entire concept. Hosted Cluster = OpenShift Cluster, Hosted Cluster is what we call a cluster created by the HyperShift Operators where the cluster's control-plane (including the API endpoint) is hosted on the management cluster. 

IF we can centralize the term

Reply:
@zanetworker I'm glad you mentioned the "API endpoint" terminology, because I honestly thought it was a little unclear, so you justified my suspicion :)

I updated the definition as follows: `An OpenShift Platform Container cluster with its control plane and API endpoint hosted on a management cluster. The hosted cluster includes the control plane and its corresponding data plane.`

I noticed that we're using the word "tenant" in a couple of places in the glossary. I updated one to "hosted", but I'm wondering if the usage on lines 18 and 39 should stay as is. What do you think?

Reply:
18, 39 look good :) 

Thanks for the update Laura :)